inlong-manager/manager-web/src/main/resources/application-prod.properties

```diff
@@ -87,6 +87,7 @@ sort.enable.zookeeper=false
 
 # cluster node timeout interval of heartbeat unit: second, the interval multiplied by 5 represents the true heartbeat timeout interval
 cluster.heartbeat.interval=6
+reset.nodeStatus.enabled=false
```

Comment:
Please add an annotation for the new param.

Reply:
Done.